duo_wordpress.php

```diff
@@ -468,6 +485,13 @@ function duo_update_mu_options() {
         else {
             $result = update_site_option('duo_xmlrpc', 'on');
         }
+        if(isset($_POST['duo_rest'])) {
+            $rest = sanitize_text_field($_POST['duo_rest']);
+            $result = update_site_option('duo_rest', $rest);
```

Comment:
@JohnMaguire i don't even believe that this plugin works on MU - so this is a moot point. I tried to test it to make sure that the settings all were correct to prevent a more than binary possibility for a response and I couldn't get it to work properly.